tests/testthat/test-nodes.R

```diff
@@ -84,7 +84,7 @@ test_that("NodeList methods work", {
 test_that("We cannot make nonsensical nodes", {
 
   expect_error(Node(id = FALSE))
-  expect_error(Node(id = 10))
+  #expect_error(Node(id = 10)) # why is this nonsensical?
```

Comment:
Originally node ids had to be strings. I think i saw in one of the constructors that numbers are allowed now? So we can remove this test